docs/platforms/javascript/guides/nuxt/manual-setup.mdx

```diff
@@ -162,15 +162,18 @@ export default defineNuxtConfig({
 ```
 
 To upload source maps, the Sentry Nuxt Module will automatically enable source map generation in your project if it is not already enabled.
-However, you need to explicitly enable source map generation on the client-side. To do this, add the following code to your Nuxt configuration:
+However, you need to explicitly enable source map generation on the client-side.
+To do this, add the following code to your Nuxt configuration:
 
 ```javascript {filename:nuxt.config.ts} {2}
 export default defineNuxtConfig({
-    sourcemap: { client: true }
+    sourcemap: { client: 'hidden' }
 });
 ```
 
-This step is necessary because Nuxt sets default values for source maps ([Nuxt docs](https://nuxt.com/docs/api/nuxt-config#sourcemap)), and the Sentry Nuxt Module keeps these settings when they are explicitly defined.
+The option `'hidden'` works like `true`, but suppresses the source map reference comments that would be added to the end of each generated file in the build output.
```

Comment:
m: I'd add a reason for why we want `hidden` over `true`, otherwise this info is just repeating someone else's docs.